deployment/network-operator/templates/role.yaml

```diff
@@ -256,3 +256,15 @@ rules:
       - patch
       - update
       - watch
+  - apiGroups:
```

Comment:
these files https://github.com/Mellanox/network-operator/blob/master/config/rbac/role.yaml and https://github.com/Mellanox/network-operator/blob/master/deploy/role.yaml should be updated too

Reply:
Thank you! Updated them too